.github/workflows/storybook.yml

```
@@ -22,9 +22,9 @@ jobs:
                   yarn build
                   yarn build-storybook --quiet
             - name: Deploy 🚀
-              uses: JamesIves/github-pages-deploy-action@3.6.2
+              uses: JamesIves/github-pages-deploy-action@v4
               with:
-                  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+                  TOKEN: ${{ secrets.WRITE_TOKEN }}
```

```suggestion
                  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
we can keep the old one since this repo is public